info/guide/guide-chapter-3.md

```diff
@@ -77,6 +77,60 @@ If the classes are meant to be used externally, and are not referenced inside th
 **Exception:**
 All variants of a `(private) val` logger should be placed at the beginning of the class (`(private) val log`, `LOG`, `logger`, etc.).
 
+#### <a name="r3.1.5"></a> 3.1.5 Order of declaration of top-level code structures
+Kotlin allows several top-level declaration types: classes, objects, interfaces, properties and functions.
+When declaring more than one class or zero classes (e.g. only functions), as per rule 2.2.1, you should document the whole file in the header KDoc.
+When declaring top-level structures, keep the following order:
+1. Top-level public constants (`const val`)
+2. Top-level public properties
+   1.1 `val`
+   1.2 `lateinit var`
+   1.3 `var`
+3. Public classes
+4. Public extension functions
+5. Public functions
+6. Private constants and properties, following same order as in 1 and 2
+7. Private classes
+8. Private extension functions
+9. Private functions
+
+**Note**:
+Extension functions, that use the class or interface declared in the same file (accept parameters of this type or return this type),
+should be declared immediately after the corresponding class or interface.
+
+Valid example:
+```kotlin
+package org.cqfn.diktat.example
+
+const val CONSTANT = 42
+
+val topLevelProperty = "String constant"
+
+interface IExample
+
+class Example : IExample
+
+fun Other.asExample(): Example { /*...*/ }
+
+private class Internal
+
+private fun Other.asInternal(): Internal
+```
+
+**Note**:
+kotlin scripts (.kts) allow arbitrary code to be placed on the top level. When writing kotlin scripts, you should first declare all properties, classes
```

Comment:
please create the following issue: 
1) If kts script contains other functions except run code, then need to wrap it in run {}

Reply:
#732